compiler/rustc_borrowck/src/diagnostics/conflict_errors.rs

```diff
@@ -1628,7 +1628,7 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
             Some((method_did, method_substs)),
         ) = (
             &self.body[loan.reserve_location.block].terminator,
-            crate::util::find_self_call(
+            rustc_const_eval::util::find_self_call(
```

Comment:
This function is not `#[inline]`, but is 21 lines so it may not have been inlined anyway.